src/hydrate/values/readEmailIfExists.test.ts

```diff
@@ -0,0 +1,86 @@
+import { describe, expect, it, vi } from "vitest";
+
+import { readEmailIfExists } from "./readEmailIfExists.js";
+
+const validAuthorPackage = {
+	author: "Author<author@package.com>",
+	email: "author@package.com",
+};
+
+const validEmailPackage = {
+	author: { email: "info@package.com", name: "Author" },
+};
+
+const mock$ = vi.fn();
+
+vi.mock("execa", () => ({
+	get $() {
+		return mock$;
+	},
+}));
+
+const mockGetNpmUserInfo = vi.fn();
+
+vi.mock("../../shared/getNpmUserInfo.js", () => ({
+	get getNpmUserInfo() {
+		return mockGetNpmUserInfo;
+	},
+}));
+
+describe("readEmailIfExists", () => {
+	it('Reads email from the package "author" field', async () => {
```

Comment:
[Nitpick] Other tests generally use lowercase, let's go with it here?

```suggestion
	it('reads email from the package "author" field', async () => {
```

This isn't a blocking request, I can always touch these up myself. 😄